src/core/components/courses/ProfileBilling.vue

```diff
@@ -589,6 +593,23 @@ export default {
       }
     };
 
+    const downloadCreditNote = async (creditNoteId) => {
+      try {
+        pdfLoading.value = true;
+        const pdf = await CourseCreditNotes.getPdf(creditNoteId);
+        downloadFile(pdf, 'avoir.pdf', 'application/octet-stream');
+      } catch (e) {
+        console.error(e);
+        if (e.status === 404) {
+          const { message } = readAPIResponseWithTypeArrayBuffer(e);
+          return NotifyNegative(message);
+        }
```

Comment:
Pourquoi ce cas different en cas de 404 ? Si l'avoir n'existe pas on a le message "not found" du coup

Reply:
Tu sais a quoi sert ce `readAPIResponseWithTypeArrayBuffer` ?